mock/mockimpl/svcimpls/mgmtimplconfig.go

```diff
@@ -164,6 +164,25 @@ func (x *mgmtImpl) handleGetAllBucketConfigs(source mock.MgmtService, req *mock.
 	}
 }
 
+func (x *mgmtImpl) handleBucketFlush(source mock.MgmtService, req *mock.HTTPRequest) *mock.HTTPResponse {
+	pathParts := pathparse.ParseParts(req.URL.Path, "/pools/default/buckets/*/controller/doFlush")
```

Comment:
We should probably also add the auth check to this... (we should've added it to handleAddBucketConfig as well, but I missed that). See handleGetAllBucketConfigs for reference.